### ISSUE TITLE
`width: fit-content` produces error `Invalid value for `width``

### ISSUE THREAD

**Issue by davidmaxwaterman** [stylelint/stylelint#5081]
> Clearly describe the bug
I expect this to pass without error or warning:
```
    width: fit-content;
```
or to be easily able to find how to make it pass (if it is somehow nonstandard/etc).

> Which rule, if any, is the bug related to?
Not sure. I just installed stylelint.

> What code is needed to reproduce the bug?

```css
div {
  width: fit-content;
}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "plugins": [
    "stylelint-csstree-validator"
  ],
  "rules": {
    "csstree/validator": true
  }
}
```

> Which version of stylelint are you using?
13.7.2

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?
```
npx stylelint *.css
```

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?
No.

> What did you expect to happen?
"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors did you get)?
```shell
✖  Invalid value for `width`   csstree/validator
```

**Comment by hudochenkov** [comment]
Open issue in https://github.com/csstree/stylelint-validator, please.
